participants/scipy_038/pr_tutorial/simple_functions.py

```diff
@@ -10,3 +10,19 @@ def factorial(value):
         return 1
     else:
         return value * factorial(value - 1)
+
```

Comment:
Love that it takes user input and displays a message, I do think it was just supposed to return a True or False lol

Reply:
I updated my code. It is 100% original and totally not stolen from someone. Trust me.

Reply:
LMAO